tests/system/Database/Live/GetVersionTest.php

```diff
@@ -29,6 +29,12 @@ final class GetVersionTest extends CIUnitTestCase
 
     public function testGetVersion(): void
     {
+        if ($this->db->DBDriver === 'MySQLi') {
```

Comment:
This should can use `if ($this->db instanceof \CodeIgniter\Database\MySQLi\Connection)`

Reply:
Yes, but since this type of comparison is widely used in our tests, I personally find it more readable. However, if others prefer the change, I'm happy to update it.

Reply:
The benefit of instanceof is phpstan will be fixed for no property notice